hack/update-bazel.sh

```diff
@@ -79,3 +79,9 @@ buildozer -quiet 'add tags manual' '//vendor/...:%go_binary' '//vendor/...:%go_t
 if [[ $ret != 0 && $ret != 3 ]]; then
   exit 1
 fi
+
+# we need to set this because gazelle doesn't support pkg-config, which would set this link option
+buildozer -quiet 'set clinkopts select({"@io_bazel_rules_go//go/platform:linux":["-lseccomp",],"//conditions:default":[],})' //vendor/github.com/seccomp/libseccomp-golang:go_default_library
```

Comment:
This is the same think we had before, modulo formatting. So far experimentation suggests that `-lseccomp` is the only link flag `pkg-config` would give us if we were using `go build` / CGO so I think this should be fine.

Without this trick here we lose the link option in the bazel build.

I don't love this, but I like it a lot more than using ancient untagged versions of libraries, and more than dealing with removing bazel or something...

It should be functionally equivalent to how we built before, just with this snippet here correcting the BUILD file manually.

Reply:
needs ret logic at EOL.

Reply:
thanks, I spent so much time futzing with the patching that I forgot this :(

Reply:
fixed